internal/kafka/internal/presenters/kafka.go

```diff
@@ -52,9 +53,17 @@ func PresentKafkaRequest(kafkaRequest *dbapi.KafkaRequest) public.KafkaRequest {
 		InstanceType:            kafkaRequest.InstanceType,
 		ReauthenticationEnabled: kafkaRequest.ReauthenticationEnabled,
 		KafkaStorageSize:        kafkaRequest.KafkaStorageSize,
+		BrowserUrl:              fmt.Sprintf("%s/%s/dashboard", validateBrowserUrl(browserUrl), reference.Id),
```

Comment:
don't need to check if it has a "/" suffix here, the `TrimSuffix()` will return the same string if it doesn't. 

```suggestion
		BrowserUrl:              fmt.Sprintf("%s/%s/dashboard", strings.TrimSuffix(browserUrl), reference.Id),
```